service/kvmconfig/v4/key/key.go

```diff
@@ -35,9 +35,11 @@ const (
 	SuccessThreshold = 1
 
 	FlannelEnvPathPrefix = "/run/flannel"
+	CoreosImageDir       = "/var/lib/coreos-kvm-images"
```

Comment:
But the images are still fetched into `/usr/code/images`. See: https://github.com/giantswarm/k8s-kvm/blob/master/docker-entrypoint.sh#L63